DirectEmployers/MyJobs
# revert changes to searchInput

## What's this PR do?

## Screenshots (if appropriate)

## Where should the reviewer start?

## How should this be manually tested?
Search input should not be cleared. 
In PRM: city field
In Analytics: the company select field.
## Any background context you want to provide?
